### PR TITLE
[TASK] code formating PSR2, removed some useless functions

### PR DESCRIPTION
**Pull request by anjeylink**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -13,6 +13,6 @@
     "typo3-ter/pxa_cookie_bar": "self.version"
   },
   "require": {
-    "typo3/cms": ">=6.2.0 <=8.9.99"
+    "typo3/cms": ">=6.1.0 <=8.9.99"
```
Comment (MattiasNilsson) [inline composer.json:16]: Why is it a drowngrade of dependency?

**Review by MattiasNilsson — COMMENTED** [review]
This looks good to me 👍 Just a minor question.

File: composer.json (hunk 1 above)
Comment (anjeylink) [reply composer.json:16]: it's working on typo3 6.1 too. but maybe 6.1 doesn't support composer mode?

Comment (MattiasNilsson) [reply composer.json:16]: No but it´s not a problem here.

**Pull request merged**